src/storage/tracer/base_memindex.cppm

```diff
@@ -35,6 +35,11 @@ protected:
         auto *memindex_tracer = InfinityContext::instance().storage()->memindex_tracer();
         memindex_tracer->AddMemUsed(mem);
     }
+
+    void DecMemUsed(SizeT mem) {
```

Comment:
How about change the function name to IncreaseMemoryUsage and DecreaseMemoryUsage